CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ You can find its changes [documented below](#060---2020-06-01).
 - `Scale::from_scale` to `Scale::new`, and `Scale` methods `scale_x` / `scale_y` to `x` / `y`. ([#1042] by [@xStrom])
 - Major rework of keyboard event handling. ([#1049] by [@raphlinus])
 - `Container::rounded` takes `KeyOrValue<f64>` instead of `f64`. ([#1054] by [@binomial0])
+- `ExtEventSink` was redone to work per-window instead of globally. ([#1137] by [@jneem])
```

Comment:
I like this and it is likely much better for most use cases, but I wonder if there is a valid need for an event sink at the delegate level? Because with this it seems no longer possible to have some application wide background thread / event sink.

Reply:
Yeah, this was my main concern too, and I was kind of hoping that someone who's using it that way would stop by and explain their use case.

I thought about trying to support both global and per-window ExtEventSinks; not only is it a bit tricky, the existing global one doesn't work that predictably/reliably for a reason that I don't know how to fix: it needs an idle loop to work, and the idle loop is tied to a window. The existing `ExtEventSink` works around this by jumping to a new window when the existing one is closed, but it makes command-targeting work strangely and I think it will also fail if all windows are closed but the application is still running.